app/jobs/meetup_member_sync.rb

```diff
@@ -0,0 +1,19 @@
+class MeetupMemberSync
```

Comment:
This job is sensible. I suggest a tweak to the naming, for it to end in `Job` to match the current naming pattern for the backend classes that are background jobs.

I'll offer `LocateUsersWithDataFromMeetupJob` unless there are thoughts about this worker being responsible other types of user data in the future.

Reply:
Possibly `MeetupMemberToUserSyncJob`?

Reply:
Maybe? That reads to me like we're creating users or something. `MeetupMemberSyncJob` could be fine if we want to keep the details of the sync out of the name.

Reply:
`LoCaTe_MemBeRsMEeTUp4MeMBErs`

Reply:
:shipit:

Reply:
Names are hard

Reply:
So this updates members if there is new meetup data?

`UpdateMembersMeetupDataJob`?

Reply:
It doesn't overwrite, it just adds any missing data from a meetup call

Reply:
I would describe the current behavior as "update a user with information about them from Meetup if they are a member of a Meetup." That information is currently location info nothing specific about Meetup, but could become more later.

Reply:
Sure, maybe `ExternalUserAccountUpdateJob`

Reply:
My vote is to add Job to the current class name and revisit the name when we want it to do more.

Reply:
Done